example/app/app.component.ts

```diff
@@ -24,6 +24,8 @@ import { Component, ViewEncapsulation } from '@angular/core';
 import { Http } from '@angular/http';
 import { Observable } from 'rxjs/Observable';
 import 'rxjs/add/observable/zip';
+import 'rxjs/add/operator/map';
```

Comment:
good catch!, actually no. `tslint` actually complains about unused imports but apparently, it doesn't check `example` folder.